src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs

```diff
@@ -395,6 +399,9 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddAutoMapper(AppDomain.CurrentDomain.GetAssemblies());
             
+            services.AddTransient<IContentThemeService, ContentThemeService>();
```

Comment:
These `Content.Services` services need to be included here in Admin because they are in use by the ContentCacheService which is currently in direct use by Admin.

When we move towards the plan in https://dfedigital.atlassian.net/browse/EES-3528, we would instead be sending a request to the Content API to update its cached items instead of it being done from Admin directly, and so these DI dependencies can then be removed along with ContentCacheService.

I can always add a TODO to this affect, referencing an existing or a new caching ticket.

Reply:
I think adding a TODO to the code would be a good idea to retain this reason for anyone looking through these services in the `Startup.cs`.